joss/paper.md

```diff
@@ -166,14 +168,14 @@ The `fitgrid` documentation is available online:
 
   ![Downloadable Examples Gallery](examples_gallery.png)
 
-## Installation, Continuous Integration, and Source 
+## Installation, Continuous Integration, and Source
 
 The online documentation includes [installation instructions]() and
-system recommendations.  The latest stable release of `fitgrid` and
+system recommendations. The latest stable release of `fitgrid` and
 the bleeding edge pre-release development version are packaged for
 Python 3.6, 3.7, and 3.8 on x86_64 linux and distributed on Anaconda
 Cloud. Installation of the stable release into a fresh conda virtual
-environment along with other compatible packages, e.g., `jupyter` for
+environment along with other compatible packages, such as `jupyter` for
 running Example Gallery notebooks, is recommended like so:
 
 ```bash
```

Comment:
why are these installation instructions different from the README? especially with regards to the conda channels.

Reply:
Yes this is a serious problem, the error is mine. To make matters worse, recent changes in upstream packages have impacted the dependencies and broke our CI. I am working on fixes for that and cleaning up the installation instructions now. Will do a patch release to 0.5.1 asap so the README, online docs, and JOSS ms installation instructions are consistent and correct. In the meantime, if you like I can post instructions for installing the pre-release version that at last check installs and runs the examples on x86_64 linux and Intel Macs.

Reply:
I see, thanks. I have opened https://github.com/kutaslab/fitgrid/issues/196 to ask some more questions about the installation. As for testing -> I'll use the stable version, which I successfully installed yesterday (but haven't looked into it more than that until now)

Reply:
Ok, for the default install you may run into issues with lmer or the parallel processing depending on what versions of what you ended up with. The pre-release is testing OK, let me know if you want to take it for a spin. Thx.